packages/app/source/client/controllers/todos-controller.js

```diff
@@ -20,23 +23,37 @@ Space.Object.extend(Todos, 'TodosController', {
     this.send(new Todos.CreateTodo({
       targetId: this.configuration.todoListId,
       title: event.title,
-      id: new Guid(),
       isCompleted: false
```

Comment:
No, it should be removed as id was removed. It and iCompleted where here because of the limitations of BDD tests at some point in time. Thanks for the note.